koji/vars/users.yml

```diff
@@ -0,0 +1,97 @@
+---
+enabled_users:
+  - name: ehelms
+    permissions:
+      - admin
+      - build
+      - repo
+  - name: evgeni
+    permissions:
+      - admin
+  - name: ewoud
+    permissions:
+      - admin
+  - name: jenkins
+    permissions:
+      - build
+      - repo
+  - name: jenkins.pulp
```

Comment:
Is this still used? Or only when Pulp 2 really is EOL?

Reply:
That was my assumption -- @pcreech do you know?

Reply:
It's the one for "nightly" builds of pulp.  Which, nightly hasn't really worked in a while anyawys.  We can probably kill 'em